AccountService.Domain/Services/JwtService.cs

```diff
@@ -36,7 +36,8 @@ public string CreateJwtToken(User user)
             {
                 Subject = new ClaimsIdentity(new Claim[]
                 {
-                    new Claim("Id",user.Id.ToString())
+                    new Claim("Id",user.Id.ToString()),
+                    new Claim("Role",user.Roles.FirstOrDefault())
```

Comment:
Is there another way to get a single user role without using collection and FirstOrDefault() method?